### PR TITLE
Say we're only accepting small bug fixes from the community

### PR DESCRIPTION
**Pull request by kienstra**
## Changes
* Says in the GitHub PR template that we're only accepting minor bug fixes from the community
* Make the issue template more clear that we're not adding features

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature request
-about: Request an enhancement, not a bug fix
+about: Thanks, but we're not adding new features
```
Comment (kienstra) [inline .github/ISSUE_TEMPLATE/feature_request.md]: It already says this on line 7 below.

But now we'd also say it before they click 'Feature request':

Screenshot 2023-12-06 at 3 30 01 PM

**Comment by kienstra** [comment]
Hi @dreamwhisper,
What do you think about this?

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (dreamwhisper) [inline .github/ISSUE_TEMPLATE/feature_request.md]: What do you think about this @kienstra?

```suggestion
about: We're currently not adding new features or enhancements, but appreciate your interest. 
```

Comment (kienstra) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Thanks, that's perfect.

**Review by dreamwhisper — APPROVED** [review]

**Comment by kienstra** [comment]
Thanks a lot, @dreamwhisper!

**Comment by henshaw** [comment]
@kienstra @dreamwhisper, I appreciate you and WP Engine keeping this plugin going, even if there may not be any plans to update it with new features. I've been using it since it was BlockLab, and it's an integral part of how my WP sites function, all of which are hosted at WP Engine.

**Comment by kienstra** [comment]
Hi @henshaw,
Wow, what an honor you've been with us since Block Lab! 

Thanks for your kind words about Genesis Custom Blocks, and how important it is with your WP Engine sites.

**Pull request merged**